index.html

```diff
@@ -9,7 +9,10 @@
 <!--This layout is not 100% right, change it if you find a better way which is more make sense-->
     <header>
         <div id="menu">Menu</div>
-        <div id="cover">Cover</div>
+        <div id="cover">
+            <h1>Biết biển rộng được bao nhiêu nếu thuyền cứ mãi bám bờ, yo</h1>
+            <div id="cover-overlay"></div>
```

Comment:
Normally, overlay should be a class thus we can reuse it in future, consider move the height out or make it match parent.